packages/phoenix-event-display/src/three/index.ts

```diff
@@ -579,4 +583,52 @@ export class ThreeManager {
   public getObjectByName(objectName: string): Object3D {
     return this.getSceneManager().getScene().getObjectByName(objectName);
   }
+
+  public addGeometryFromParameters(parameters: any): void {
```

Comment:
Can you quickly add docs to this function? Or may be do it in another PR or commit.
We should have the documentation to 100% for the event display. :)

Reply:
Good point! Thanks! 

I actually want to change `any` to specify the required parameters... but the main thing was to get this working before the presentation next week.